docs/modules/hivemq.md

```diff
@@ -0,0 +1,239 @@
+# HiveMQ Module
+
+<img src="../modules_logos/hivemq-testcontainer.png" alt="drawing" width="400"/>
+
+Automatic starting HiveMQ docker containers for JUnit4 and JUnit5 tests.
+This enables testing MQTT client applications and integration testing of custom HiveMQ extensions.
+
+- Community forum: https://community.hivemq.com/
+- HiveMQ website: https://www.hivemq.com/
+- MQTT resources:
+    - [MQTT Essentials](https://www.hivemq.com/mqtt-essentials/)
+    - [MQTT 5 Essentials](https://www.hivemq.com/mqtt-5/)
+    
+Please make sure to check out the hivemq-docs for the [community edition](https://github.com/hivemq/hivemq-community-edition/wiki/) 
+and the [regular edition](https://www.hivemq.com/docs/hivemq/4.7/user-guide/).
+
+## Add to your project
+
+### Gradle
+
+Add to `build.gradle`:
+
+````groovy
+testImplementation 'org.testcontainers:hivemq:{{latest_version}}'
+````
+
+Add to `build.gradle.kts`:
+
+````kotlin
+testImplementation("org.testcontainers:hivemq:{{latest_version}}")
+````
+
+### Maven
+
+Add to `pom.xml`:
+
+```xml
+<dependency>
+    <groupId>org.testcontainers</groupId>
+    <artifactId>hivemq</artifactId>
+    <version>{{latest_version}}</version>
+    <scope>test</scope>
+</dependency>
+```
+
+
+## Using HiveMQ CE/EE
+
+We provide different editions of HiveMQ on [Docker-Hub](https://hub.docker.com/u/hivemq):
+
+- the open source [Community Edition](https://github.com/hivemq/hivemq-community-edition) which 
+is tagged as *hivemq/hivemq-ce*.
+- the Enterprise Edition which is tagged as *hivemq/hivemq-ee*.
```

Comment:
'Tagged' is a bit of a loaded term in Docker; perhaps instead it could be:
```suggestion
- the open source [Community Edition](https://github.com/hivemq/hivemq-community-edition) which 
is published as *hivemq/hivemq-ce*.
- the Enterprise Edition which is published as *hivemq/hivemq-ee*.
```